Clarify how infections are estimated (#571)
epiforecasts/EpiNow2: README.Rmd

<details> <summary> More details </summary>

> `{EpiNow2}` estimates the time-varying reproduction number on cases by date of infection (using a similar approach to that implemented in [`{EpiEstim}`](https://github.com/mrc-ide/EpiEstim)). True infections treated as latent and unobserved, estimated, and then mapped to observed data (for example cases by date of report) via a series of uncertain delay distributions (in the examples in the package documentation these are an incubation period and a reporting delay) and a reporting model that can include weekly periodicity. 

Reviewer: ```suggestion
`{EpiNow2}` estimates the time-varying reproduction number on cases by date of infection (using a similar approach to that implemented in [`{EpiEstim}`](https://github.com/mrc-ide/EpiEstim)). True infections treated as latent and unobserved, are estimated and then mapped to observed data (for example cases by date of report) via one or more delay distributions (in the examples in the package documentation these are an incubation period and a reporting delay) and a reporting model that can include weekly periodicity. 
```

Author: I actually wanted to say "True infections [ARE] treated as latent and unobserved, estimated, and ..."

What do you think?
